test/Microsoft.AspNet.Mvc.IntegrationTests/ArrayModelBinderIntegrationTest.cs

```diff
@@ -160,8 +160,8 @@ public class ArrayModelBinderIntegrationTest
             var modelBindingResult = await argumentBinder.BindModelAsync(parameter, modelState, operationContext);
 
             // Assert
-            Assert.NotNull(modelBindingResult);  // This fails due to #1579
-            Assert.False(modelBindingResult.IsModelSet);
+            Assert.NotNull(modelBindingResult);
+            Assert.True(modelBindingResult.IsModelSet);
```

Comment:
This was a mistake in writing these tests originally. `modelBindingResult.IsModelSet == true` is consistent with the result when an empty POCO is bound.

I did a CTRL+F for #1579 in the code and was able to enable all of them.